chapters/1.md

```diff
@@ -3,37 +3,63 @@
 
-In order to help mitigate such risks, the traditional banking system can utilize a new technology called the blockchain. A blockchain is a distributed database that continuously adds to a record of all the transactions that have occurred within a network. This database is often referred to as a ledger, because the data contains a large list of transactions that have taken place.
+It's important to remember, that when you deposit money in the bank, you don't really just have money sitting in an account with your name on it. Instead, what you really have done is *lent* money to the bank. They *owe* it to you. This is what is called a *liability*. They are allowed to use your funds for other things that the bank would like to do (like making money via lending or for investments). They are required to give it back when you ask, but the money really just exists as an IOU on some database inside of their bank. So, because Leo and George have the same bank, the bank can simply change Leo's IOU to say that they owe him $2500 less, and George's IOU that they owe him $2500 more. (They debit Leo's account and subsequently credit George's account.) 
 
-The first distributed blockchain was conceptualized in 2008 by an anonymous person or group known as Satoshi Nakamoto and implemented in 2009 as a core component of Bitcoin, where it serves as the public ledger for all transactions. The invention of the blockchain made Bitcoin the first digital currency to solve the double spending problem without the need for a trusted authority or central server.
+Both Leo and George have to trust the bank will be honest and will give them their money when they ask and will not go out of business or bankrupt while holding onto their money. Many countries have deposit insurance institutions that may be government run, or a part of the central bank, or even private institutions (one example would be the FDIC in the US). Deposit insurance allows banks to lend out or invest some of the money that they hold and not have to keep the full balance on hand, because if something bad happens where they lose the money, or they go bankrupt, the deposit insurance will make sure that everyone still gets their money back from the bank; this is called *fractional-reserve banking* and it is what is used over most of the world. So, Leo and George can trust their bank as long as they trust the government or institution that insures the money in the bank. In some places, under corrupt, or inefficient governments, this could be a bad idea, but non-the-less they have no choice if they wish to move money across large distances in the traditional banking system they currently have.
```

Comment:
FDIC in the US => Federal Deposit Insurance Corporation in the United States

Reply:
Yes, FDIC insures deposits in the US. In the EU Directive 94/19/EC requires some kind of deposit insurance but every country does it different. I'm not familiar with the European banking system though.